src/sql/workbench/contrib/notebook/browser/cellViews/code.component.ts

```diff
@@ -363,4 +375,8 @@ export class CodeComponent extends AngularDisposable implements OnInit, OnChange
 		}
 		this._editor.setHeightToScrollHeight(false, isCollapsed);
 	}
+
+	public deltaDecorations(newDecorationRange: any, oldDecorationRange: any): void {
```

Comment:
Doesn't matter too much since you're not implementing it - but shouldn't these have some actual type besides any?

Reply:
NotebookRange is the type, I didn't want to add the reference when I know for sure they aren't going to be implemented.

Reply:
This should really be implemented in the base class with a default of doing nothing. That'd be a much cleaner way of solving this.